code/modules/sealant_gun/sealant_gun.dm

```diff
@@ -58,7 +58,7 @@
 	. = ..()
 	if(loc == user)
 		if(loaded_tank)
-			. += SPAN_NOTICE("The loaded tank has about [loaded_tank.foam_charges] liter\s of sealant left.")
+			. += SPAN_NOTICE("The loaded tank has about [REAGENT_VOLUME(loaded_tank.reagents, /decl/material/liquid/foam) || 0] liter\s of sealant left.")
```

Comment:
i know it's mostly like, opaque and not exposed to the user, but 1u = 1L is a new level of unit nonsense